nautobot/core/templates/inc/search_panel.html

```diff
@@ -1,42 +1,27 @@
 {% load form_helpers %}
```

Comment:
Is it risky to make changes to this file? Another option is to simply hardcode this logic into the file `filter form modal.html.`

Reply:
I think it's OK to touch this file. I'm not aware of any plugins that are explicitly using it; normally it's just implicitly included through `generic/object_list.html`. I do see a few templates other than `object_list.html` that are using it in core though - would be good to confirm that they're all correctly rendering after this changeset and #2617:

```
nautobot/core/templates/generic/object_list.html:                {% include 'inc/search_panel.html' %}
nautobot/dcim/templates/dcim/connections_list.html:            {% include 'inc/search_panel.html' %}
nautobot/dcim/templates/dcim/rack_elevation_list.html:            {% include 'inc/search_panel.html' %}
nautobot/extras/templates/extras/plugins_list.html:                    {% include 'inc/search_panel.html' %}
```